LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Lets remove the file extension from this. Also it needs to be Apache 2.0.

You can use this as a reference: https://github.com/puppetlabs/puppet-lint-i18n/blob/master/LICENSE